data/net/minecraft/data/worldgen/placement/OrePlacements.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/data/worldgen/placement/OrePlacements
+	METHOD commonOrePlacement (ILnet/minecraft/world/level/levelgen/placement/PlacementModifier;)Ljava/util/List;
+		ARG 0 count
+		ARG 1 modifier
```

Comment:
```suggestion
		ARG 1 heightRange
```
while it could be an arbitrary modifier, in almost all usages this is some kind of height restriction, mostly commonly a `HeightRangePlacement`.

Reply:
For ores, it probably would be a height restriction.